_includes/css/agency.css

```diff
@@ -494,11 +494,10 @@ section h3.section-subheading {
     width: 80px;
     height: 80px;
     margin-left: 0;
-    border: 7px solid #f1f1f1;
     border-radius: 100%;
     text-align: center;
     color: #fff;
-    background-color: #{{ site.data.template.color.primary }};
+    filter: drop-shadow(0 0 5px #000000);
```

Comment:
What's this part about? 😮

Reply:
It'll give an effect like the circular plate even it's not in a circular shape.